credentials/apps/credentials_theme_philu/templates/philu/credentials/programs/professional-certificate/certificate.html

```diff
@@ -0,0 +1,59 @@
+<link rel="icon" type="image/x-icon" href="https://static.philanthropyu.org/favicon.png" />
+
+<main class="accomplishment-main specialization-certificate">
+    <div class="accomplishment">
+        <div class="accomplishment-rendering">
+            <div class="certificate-stamp">
+                <img src="http://static.philanthropyu.org/specialization/specialization-certificate-stamp.png" alt="">
+            </div>
+            <div class="stamp-light-bg">
+                <img src="https://static.philanthropyu.org/philanthropyu-icon-black.svg" alt="">
+            </div>
+            <div class="accomplishment-statement">
+                <h3 class="accomplishment-title">Specialization certificate</h3>
+                <h5 class="accomplishment-course-date">Awarded on {{ user_credential.modified|date:'F d, Y' }} to</h5>
+                <h1 class="accomplishment-recipient">{{ user_data.name }}</h1>
+                <h5 class="clr-bluish-grey">For successfully completing A specialization in</h5>
+                <h4 class="accomplishment-course-name">{{ user_credential.credential.program_details.title }}</h4>
+                <h5 class="accomplishment-course-detail">That includes {{ user_credential.credential.program_details.course_count|apnumber }} courses</h5>
+                <ul class="accomplishment-course-list">
+                    {% for course in user_credential.credential.program_details.courses %}
+                        <li>{{ course.title }}</li>
+                    {% endfor %}
+                </ul>
+                <p class="accomplishment-course-summary">
+                    {{ user_credential.credential.program_details.subtitle }}
+                </p>
+            </div>
+            <ul class="accomplishment-signatories">
+
+                {% for signatory in user_credential.credential.signatories.all %}
+                    <li class="accomplishment-signatory">
+                        <h4 class="signatory-sign">{{ signatory.name }}</h4>
+                        <h4 class="signatory-name">{{ signatory.name }}</h4>
+                        <h5 class="role-organization">
+                            {{ signatory.title }}
+                            {% if signatory.organization_name_override %}
+                                , {{ signatory.organization_name_override }}
+                            {% endif %}
+                        </h5>
+                        <img class="brand-logo" src="{{MEDIA_URL}}{{ signatory.image }}" alt="Philanthropy University"/>
+                    </li>
+                {% endfor %}
+            </ul>
+            <span class="disclaimer">
+                Philanthropy University is a non-degree granting initiative of Philanthropy U. Learners are not entitled to earn college or other academic credit.
+            </span>
+        </div>
+    </div>
+</main>
+<script>
+    // This line is to hide a div in base html to avoid core change.
```

Comment:
When we have made our own theme directory can't we override base.html?

Reply:
This is how we can integrate our theme with openedx, which allows us to override header, footer and main body not base.html. The only small sharing block in base.html is not customizable, So to avoid core change I hide this using javascript. 
https://edx-credentials.readthedocs.io/en/latest/theming.html